var/spack/repos/builtin/packages/biopieces/package.py

```diff
@@ -64,7 +64,7 @@ class Biopieces(Package):
     depends_on('ruby-rubyinline')
     depends_on('ruby-terminal-table')
 
-    depends_on('python@2:6', type=('build', 'run'))
+    depends_on('python@2.6;', type=('build', 'run'))
```

Comment:
I think you'd want that to be a `:` vs. a `;`

Reply:
you're right. I'm struggling today